pkg/storage/tableland/store.go

```diff
@@ -60,17 +48,58 @@ var (
 			unique(rig_id, trait_type)
 		)`,
 	}
+	// LookupsDefinition defines the lookups table.
+	LookupsDefinition = TableDefinition{
+		Prefix: "lookups",
+		Schema: `(
+			renders_cid text,
+			layers_cid text,
+			image_full_name text,
+			image_full_alpha_name text,
+			image_medium_name text,
+			image_medium_alpha_name text,
+			image_thumb_name text,
+			image_thumb_alpha_name text,
+			animation_base_url text
+		)`,
+	}
```

Comment:
@sanderpick I want to get some quick and early thumbs up or down from you about this change to the rigs tables design. To summarize, above you can see the removal of the `rigs` table, and the creation of a new `lookups` table. This is a more normalized design that optimized more for data compactness, cost savings and ease of updates. I'll leave another comment in the file showing how the metadata query changes accordingly.